tests/ui/implied-bounds/bevy_world_query.rs

```diff
@@ -19,7 +19,9 @@ impl<Q: WorldQuery + 'static> SystemParam for Query<Q> {
 
 pub struct ParamSet<T: SystemParam>(T) where T::State: Sized;
 
-fn handler<'a>(_: ParamSet<Query<&'a u8>>) {}
+fn handler<'a>(x: ParamSet<Query<&'a u8>>) {
+    let _: ParamSet<_> = x;
```

Comment:
Is this enough to reproduce the bevy regression?

Reply:
yes

Reply:
makes sense